src/client/app/components/unit/EditUnitModalComponent.tsx

```diff
@@ -103,15 +103,16 @@ export default function EditUnitModalComponent(props: EditUnitModalComponentProp
 	const shouldUpdateUnit = (): boolean => {
 		// true if inputted values are okay and there are changes.
 		let inputOk = true;
+		const meterDataByID = selectMeterDataById(store.getState())
```

Comment:
Curious why this one uses store directly and not useAppSelector as in somw other places.

Reply:
Good catch! This is bad practice!! It is higly advised against to import store.ts anywhere other than index. Honeslty not sure how this one made it through... Definitely needs to be addressed